crates/cli/src/subcommands/energy.rs

```diff
@@ -23,12 +24,7 @@ fn get_energy_subcommands() -> Vec<clap::Command> {
                     "The identity to check the balance for. If no identity is provided, the default one will be used.",
                 ),
             )
-            .arg(
-                Arg::new("server")
-                    .long("server")
-                    .short('s')
-                    .help("The nickname, host name or URL of the server from which to request balance information"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.